vars/mtaBuild.groovy

```diff
@@ -47,7 +47,9 @@ import static com.sap.piper.Utils.downloadSettingsFromUrl
      */
     'platform',
     /** Path or url to the mvn settings file that should be used as project settings file.*/
-    'projectSettingsFile'
+    'projectSettingsFile',
+    /** The name of the generated mtar file.*/
+    'mtarName'
```

Comment:
I have thought so but the existing properties were not sorted properly. I assumed that it might be intentional so kept it at last. Check the new position and let me know if it is fine.